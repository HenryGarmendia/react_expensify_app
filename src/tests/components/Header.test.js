import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/header';

test('should render header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.find('h1').text()).toBe('Expensify Web-App');
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
    // console.log(renderer.getRenderOutput());
});