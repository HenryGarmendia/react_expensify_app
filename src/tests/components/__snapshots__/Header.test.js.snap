// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    Expensify Web-App
  </h1>
  <NavLink
    activeClassName="is_active link"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is_active link"
    ariaCurrent="true"
    to="/create"
  >
    Create Expense
  </NavLink>
  <button
    onClick={[Function]}
  >
    Logout
  </button>
</header>
`;
